webcentral-backend-webapp/src/main/java/hu/bme/sch/bss/webcentral/controller/videoportal/VideoController.java

```diff
@@ -0,0 +1,31 @@
+package hu.bme.sch.bss.webcentral.controller.videoportal;
+
+import hu.bme.sch.bss.webcentral.VideoService;
+import hu.bme.sch.bss.webcentral.domain.CreateVideoRequest;
+import hu.bme.sch.bss.webcentral.model.Video;
+
+import javax.validation.Valid;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/video")
+public class VideoController {
+
+    private final VideoService videoService;
+
+    public VideoController(final VideoService videoService) {
+        this.videoService = videoService;
+    }
+
+    @PostMapping(produces = "application/json")
+    public final Video createVideo(@Valid @RequestBody final CreateVideoRequest request) {
```

Comment:
I would not return with the Database entity itself, as it expose the structure of database. If you create an explicit response object (which can be the same now as the database entity itself), then the database structure change would not effect directly the contract of you service.

Reply:
I'll fix this in [#48](https://redmine.bsstudio.hu/issues/48)